tests/Endpoints/SearchTest.php

```diff
@@ -854,4 +854,26 @@ public function testSearchAndRetrieveFacetStats(): void
 
         self::assertSame(['info.reviewNb' => ['min' => 50.0, 'max' => 1000.0]], $response->getFacetStats());
     }
+
+    public function testSearchWithDistinctAttribute(): void
```

Comment:
How does this test correlate with added distinct prop to `SearchQuery`? because I don't see anywhere using the SearchQuery :)

Reply:
let me add another test by using `SearchQuery` as well